src/fespace/fefunction.jl

```diff
@@ -117,7 +117,8 @@ function FEFunction(
     ::Type{<:Lagrange},
     feSpace::AbstractFESpace,
     mesh::AbstractMesh,
-    f::AbstractLazy,
+    f::AbstractLazy;
+    backend::BcubeBackendCPUSerial = get_bcube_backend(),
```

Comment:
here again, maybe this function should take a `Domain` as an input, and `Domain` could have the backend